### PR TITLE
[ADD] website_crm_address

### PR DESCRIPTION
**Pull request by hbrunn**
# 
# Address in contact page

This module was written to allow visitors to leave their address on the contact page, so that partners created from the resulting lead already have the correct address set.
# Usage

Fill in the additional fields on the website's contact form.

### CODE REVIEW

**Comment by rafaelbn** [comment]
Tested in runbot :+1: 
Only one remark I think as "company"in core, "address" added by this module must be optional (see screenshoot)

**Comment by hbrunn** [comment]
well, it's optional as in: If you want the fields, install the module, if not, don't

**Comment by whoople** [comment]
@hbrunn :
If you don't consider **website_multi**, you're right...
Cheers.

File: website_crm_address/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+##############################################################################
+from openerp import http
+from openerp.addons.website_crm.controllers.main import contactus
+
+
+class ContactUs(contactus):
+    @http.route(['/crm/contactus'])
+    def contactus(self, **kwargs):
+        if 'zipcode' in kwargs:
+            kwargs['zip'] = kwargs['zipcode']
```
Comment (yajo) [inline website_crm_address/controllers/main.py:28]: Why not just call the HTML input "zip" and avoid controllers?

**Comment by rafaelbn** [comment]
Hi @hbrunn , please answer @Yajo 's question and I think we wil test and merge this module. After this we will improve this module adding what I said here https://github.com/OCA/website/pull/77#issuecomment-139913883 to support multiwebsite. OK? Thanks!

**Comment by rafaelbn** [comment]
Please @hbrunn could you rebase and answer @Yajo ? If you don't have time let us know we will finish this PR

@dgzurita @JavierIniesta please review this PR please, thanks!

File: website_crm_address/controllers/main.py (hunk 1 above)
Comment (hbrunn) [inline website_crm_address/controllers/main.py:28]: `zip` is a python builtin, and if you do what you propose, you get the function reference as value

**Comment by hbrunn** [comment]
@rafaelbn done

**Comment by rafaelbn** [comment]
This PR is ready to be merged :+1:

**Comment by antespi** [comment]
Website options are not multi-website supported yet, even installing odoo-extra/website_multi addon.
You can talk about it here: https://github.com/odoo/odoo/issues/9745

**Comment by dgzurita** [comment]
Tested in runbot :+1:

**Comment by rafaelbn** [comment]
@Yajo This PR is related with https://github.com/OCA/website/tree/8.0/website_snippet_contact_form

**Comment by JavierIniesta** [comment]
Tested in runbot :+1:

**Pull request merged**